src/Esprima/JsxParser.cs

```diff
@@ -394,7 +394,7 @@ private string ScanXHTMLEntity(char quote)
             else if (hex && str.Length > 2)
             {
                 result =
-                    ((char) int.Parse(str.Substring(2), System.Globalization.NumberStyles.HexNumber)).ToString();
+                    ((char) int.Parse(str.Substring(2), System.Globalization.NumberStyles.AllowHexSpecifier)).ToString();
```

Comment:
probably not needed for this PR, but we also have `HAS_SPAN_PARSE` defined when can use span based parsing, some linting tools will also probably warn about not using invariant culture

Reply:
I'll take care of this and similar minor issues in a separate PR.